src/deployment/cd.md

```diff
@@ -231,7 +231,7 @@ Create a file at `ios/ci_scripts/ci_post_clone.sh` and add the content below.
 cd $CI_WORKSPACE # change working directory to the root of your cloned repo.
 
 # Install Flutter using git.
-git clone https://github.com/flutter/flutter.git -b stable $HOME/flutter
+git clone https://github.com/flutter/flutter.git --depth 1 -b stable $HOME/flutter
```

Comment:
This is snippet'd in from `deployment/xcode_cloud/ci_post_clone.sh`. Please modify the source and use the tool referenced in the README to update the snippets. The CI will fail this as it stands.

Reply:
I think it should work proper now (the fun times trying to run the tooling under windows, ha).